### ISSUE TITLE
VGG pretrained

### ISSUE THREAD

**Issue by cirobr** [FluxML/Metalhead.jl#268]
Cheers. Docs from stable v0.9.1 shows VGG as pretrained model, however function call does not have a `pretrain` argument as, for instance, ResNet. Can you please clarify if VGG is loaded with pretrained weights? 

If positive, please kindly inform which dataset is used for training, and if any of the convnet configurations from [article](https://arxiv.org/pdf/1409.1556.pdf) are pretrained (page 3).

Thanks in advance.

**Comment by theabhirath** [comment]
Hi, I think the docs are glitching because there are two docstrings for the same name, and only one seems to be displayed! VGG does in fact have pretrained weights and a function API that exposes them, you can see the docstring for that here: https://github.com/FluxML/Metalhead.jl/blob/2ad3337e42a8df03a411f2c38b83028c68cb8286/src/convnets/vgg.jl#L144-L160
As for the configurations, I could have sworn we had a note showing that, but it probably got removed somewhere by accident. The dataset used for training is ImageNet, and VGG 11, 13, 16 and 19 all have pre-trained weights, albeit when the models are loaded without `BatchNorm` (this is mirroring the paper, which does not have `BatchNorm`, but PyTorch does provide this option which we do not yet have).

**Comment by cirobr** [comment]
Cheers, have checked if Metalhead v0.9.1 is installed, then called a model such as `VGG(19; pretrain=true)`. It works. Thanks.
